src/main/resources/application.yaml

```diff
@@ -23,19 +23,26 @@ service:
   storage-config:
     - name: bulkscan
       sas-validity: 300 #In seconds
+      blob-destination: bulkscan
```

Comment:
not sure blob-destination would be a proper name as it would be mean blob is getting copied to bulkscan but actually it would be cmc or probate and so on. We can rename it later not required for this PR

Reply:
how about `target-storage-account`? Or  `storage-account-desination`?